Packs/CortexResponseAndRemediation/Playbooks/playbook-Uncommon_execution_of_ODBCConf.yml

```diff
@@ -0,0 +1,715 @@
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "4":
+    id: "4"
+    taskid: 765fe8d2-bdd1-4be4-8a98-48c82c984a70
+    type: title
+    task:
+      id: 765fe8d2-bdd1-4be4-8a98-48c82c984a70
+      version: -1
+      name: Remediation
+      description: Terminate a process tree by its causality ID. Available only for XSIAM 2.4.
+      type: title
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "10"
+    separatecontext: false
+    continueonerror: true
+    continueonerrortype: errorPath
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 1170
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "5":
+    id: "5"
+    taskid: bd9cacdf-4ffb-44e7-81b4-7d958cb76986
+    type: regular
+    task:
+      id: bd9cacdf-4ffb-44e7-81b4-7d958cb76986
+      version: -1
+      name: Close Alert
+      description: Close the current alert as True Positive.
+      script: Builtin|||closeInvestigation
+      type: regular
+      iscommand: true
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "6"
+    scriptarguments:
+      closeNotes:
+        simple: Malicious activity detected - Alert was remidated
+      closeReason:
+        simple: Resolved - Handled by the playbook "Suspicious execution of ODBCConf"
+      id:
+        complex:
+          root: alert
+          accessor: id
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 1700
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "6":
+    id: "6"
+    taskid: deab0d59-be1c-454b-8043-540b7456529e
+    type: title
+    task:
+      id: deab0d59-be1c-454b-8043-540b7456529e
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 1870
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "7":
+    id: "7"
+    taskid: 8d7bf580-9887-46c9-85bc-05eab9fad48f
+    type: regular
+    task:
+      id: 8d7bf580-9887-46c9-85bc-05eab9fad48f
+      version: -1
+      name: Terminate Process Manually
+      description: |-
+        Dear Analyst,
+
+        During the remediation process, the playbook couldn’t terminate the process: ${alert.cgoname}
+
+        Please terminate the process manually if possible.
```

Comment:
Can we provide more data such as the name of the process we tried to kill?

Reply:
the cgoname is displayed in the line above this one...